nixos/tests/installer.nix

```diff
@@ -75,7 +75,7 @@ let
     else ''
       def assemble_qemu_flags():
           flags = "-cpu max"
-          ${if system == "x86_64-linux"
+          ${if (system == "x86_64-linux" || system == "i686-linux")
             then ''flags += " -m 1024"''
             else ''flags += " -m 768 -enable-kvm -machine virt,gic-version=host"''
```

Comment:
`gic-version` smells like an ARM-specific change.

So I think the change is probably fine here.

Though I'm wondering why on ARM 768MiB of RAM is used vs 1024 on x86. (Not relevant to these changes)